<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutForm</class>
 <widget class="QWidget" name="AboutForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">About 3DCityDB Tools</string>
  </property>
  <widget class="QListWidget" name="listMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>121</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="autoScrollMargin">
    <number>15</number>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="gridSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="itemAlignment">
    <set>Qt::AlignLeading</set>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>About the plug-in</string>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Developers</string>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Changelog</string>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>References</string>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>License</string>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Get the 3DCityDB!</string>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedContents">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>551</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pgAbout">
    <widget class="QTextBrowser" name="txtAbout">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>371</height>
      </rect>
     </property>
     <property name="markdown">
      <string notr="true"/>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIssueBug">
     <property name="geometry">
      <rect>
       <x>376</x>
       <y>390</y>
       <width>165</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Issue a bug</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenGitHub">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>165</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Open GitHub repository</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIdeas">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>390</y>
       <width>165</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Ideas, suggestions?</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pgDevelopers">
    <widget class="QTextBrowser" name="txtDevelopers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pgChangelog">
    <widget class="QTextBrowser" name="txtChangelog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pgReferences">
    <widget class="QTextBrowser" name="txtReferences">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>401</height>
      </rect>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="openLinks">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pgLicense">
    <widget class="QTextBrowser" name="txtLicense">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>401</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>-9</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="documentTitle">
      <string notr="true"/>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::WidgetWidth</enum>
     </property>
     <property name="lineWrapColumnOrWidth">
      <number>0</number>
     </property>
     <property name="markdown">
      <string notr="true"/>
     </property>
     <property name="html">
      <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
     <property name="searchPaths">
      <stringlist notr="true"/>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="openLinks">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pgEmpty"/>
   <widget class="QWidget" name="pg3DCityDB">
    <widget class="QTextBrowser" name="txt3DCityDB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>371</height>
      </rect>
     </property>
     <property name="documentTitle">
      <string notr="true"/>
     </property>
     <property name="markdown">
      <string notr="true"/>
     </property>
     <property name="html">
      <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3DCityDBDownload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>165</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Download 3DCityDB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3DCityDBManual">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>390</y>
       <width>165</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>3DCityDB manual</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3DCityDBInstall">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>390</y>
       <width>165</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>3DCityDB install guide</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>569</x>
     <y>520</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_img">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../icons/plugin_icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_txt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>3DCityDB Tools for QGIS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listMenu</tabstop>
  <tabstop>btnOpenGitHub</tabstop>
  <tabstop>btnIdeas</tabstop>
  <tabstop>btnIssueBug</tabstop>
  <tabstop>txtAbout</tabstop>
  <tabstop>txtDevelopers</tabstop>
  <tabstop>txtChangelog</tabstop>
  <tabstop>txtReferences</tabstop>
  <tabstop>txtLicense</tabstop>
  <tabstop>txt3DCityDB</tabstop>
  <tabstop>btn3DCityDBInstall</tabstop>
  <tabstop>btn3DCityDBManual</tabstop>
  <tabstop>btn3DCityDBDownload</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
